cts/scheduler/summary/leftover-pending-monitor.summary

```diff
@@ -12,16 +12,13 @@ Current cluster status:
       * Stopped: [ node-1 node-2 ]
 
 Transition Summary:
-  * Stop       stateful-1:1     ( Promoted node-3 )  due to unrunnable stateful-1:0 monitor
-  * Start      stateful-1:2     (          node-1 )  due to unrunnable stateful-1:0 monitor (blocked)
+  * Start      stateful-1:2     ( node-1 )
```

Comment:
I think this start should still be blocked. If we don't know the state on all nodes, we shouldn't risk starting too many instances. I'm not sure offhand how to accomplish that. Maybe for clone instances, instead of ordering relative to the clone start/reload, we should order the instance's probe before each inactive instance's start/reload.